lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -404,6 +414,79 @@ spec = do
         r <- joinStakePool ctx (p ^. #id) (w, "Secure Passprase")
         expectResponseCode HTTP.status404 r
 
+    it "STAKE_POOLS_ESTIMATE_FEE_01 - \
+        \wallet with [fee - 1] => cannot estimate fee" $ \ctx -> do
+        putStrLn "hello"
+        (_, p:_) <- eventually $
+            unsafeRequest @[ApiStakePool] ctx listStakePoolsEp Empty
+        fee <- do
+            w <- fixtureWallet ctx
+            (_, r) <- joinStakePoolFee ctx (p ^. #id) w
+            print r
+            return $ either (error . show) (getQuantity . amount) r
+
+        w <- fixtureWalletWith ctx [fee - 1]
+        joinStakePoolFee ctx (p ^. #id) w >>= flip verify
+            [ expectResponseCode HTTP.status403
+            , expectErrorMessage "I'm unable to select enough coins to pay for a"
```

Comment:
**[change required]**

Use `errMsg403DelegationFee` from `Test.Integration.Framework.TestData`

**Why?**

Avoid magic constants in code.